sensor/common/registry/lazy_tls_image_registry_test.go

```diff
@@ -36,9 +37,10 @@ func TestNoPanics(t *testing.T) {
 func TestConfig(t *testing.T) {
 	ctrl := gomock.NewController(t)
 	mockReg := regMocks.NewMockRegistry(ctrl)
+	ctx := context.Background()
 	creator := func(integration *storage.ImageIntegration, options ...types.CreatorOption) (types.Registry, error) {
 		cfg := integration.GetDocker()
-		mockReg.EXPECT().Config().Return(&types.Config{
+		mockReg.EXPECT().Config(ctx).Return(&types.Config{
```

Comment:
Is the context actually relevant in the test ? Could we expect `gomock.Any()` instead or do we want to make sure that the context is not modified by calls to `Config(ctx)`?

Reply:
I don't think the context is relevant here, I was just trying to make the tests pass.